JincResize/JincResize.cpp

```diff
@@ -8,8 +8,47 @@
 #include "VapourSynth.h"
 #include "VSHelper.h"
 
-#define JINC_ZERO 1.2196698912665045
-#define M_PI      3.14159265358979323846
+constexpr double JINC_ZERO_SQR = 1.48759464366204680005356;
+#ifndef M_PI // GCC seems to have it
+constexpr double M_PI = 3.14159265358979323846;
+#endif
+
+// Doesn't double precision overkill?
```

Comment:
I compared [my float implementation](https://github.com/luglio/VapourSynth-JincResize/blob/AVX2/JincResize/JincResize.cpp#L70) and the double one. The difference of the outputs is very likely as small as 2 in 16-bit integer format, which I think could be neglected. I'm kind of lazy to change the pasted constants to float since compilers will convert them.
BTW I guess in this version the header `new` is redundant.

Reply:
I also think `new` is redundant. It is about "low level memory management". These functions are not used in this version.